addons/binding/org.openhab.binding.onkyo/src/main/java/org/openhab/binding/onkyo/OnkyoBindingConstants.java

```diff
@@ -80,7 +82,7 @@
     public static final Set<ThingTypeUID> SUPPORTED_THING_TYPES_UIDS = Stream
             .of(THING_TYPE_ONKYOAV, THING_TYPE_ONKYO_UNSUPPORTED, THING_TYPE_TXNR414, THING_TYPE_TXNR515,
                     THING_TYPE_TXNR525, THING_TYPE_TXNR535, THING_TYPE_TXNR555, THING_TYPE_TXNR616, THING_TYPE_TXNR626,
-                    THING_TYPE_TXNR646, THING_TYPE_TXNR656, THING_TYPE_TXNR717, THING_TYPE_TXNR727, THING_TYPE_TXNR737,
+                    THING_TYPE_TXNR646, THING_TYPE_TXNR656, THING_TYPE_TXNR708 ,THING_TYPE_TXNR717, THING_TYPE_TXNR727, THING_TYPE_TXNR737,
```

Comment:
The space should be added after the comma and not in front of it. ;-)

Reply:
Please also add this new emodel also to the Discovery section of the Onkyo README.md